PCU/src/main/java/com/coffeelovers/pcu/controller/RawMaterialController.java

```diff
@@ -105,7 +105,7 @@
 	}
 	
 	@PostMapping("/rawMaterials")
-	public ResponseEntity<RawMaterial> createRawMaterial(@RequestBody RawMaterial rawMaterial){
+	public ResponseEntity<InvoiceDetails> createRawMaterial(@RequestBody InvoiceDetails rawMaterial){
```

Comment:
the variable name here too

Reply:
No need. it is already done on my side. ignore it please